src/init.cpp

```diff
@@ -295,7 +295,7 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-conf=<file>", strprintf(_("Specify configuration file (default: %s)"), "bitcoin.conf"));
     if (mode == HMM_BITCOIND)
     {
-#if !defined(WIN32)
+#ifndef WIN32
```

Comment:
The other six usages are `#ifndef` and it was a different author who chose the `#if !defined()` twice. But I can remove those if you prefer less diff over consistency.